bundler.go

```diff
@@ -483,8 +495,11 @@ func (b *Bundler) provisionVendor(oS, arch string) (err error) {
 		return
 	}
 
+
+	astilog.Debugf("provisionVendor b.versionElectron %s", b.versionElectron)
+
 	// Provision electron
-	if err = b.provisionVendorElectron(oS, arch); err != nil {
+	if err = b.provisionVendorElectron(oS, arch, b.versionElectron); err != nil {
```

Comment:
You shouldn't need to inject this parameter since `provisionVendorElectron` is a `Bundler` method and therefore has already the value of this parameter in its struct

Reply:
Fixed to use versions stored in Bundler.